### ISSUE TITLE
Disable job_office model in greenhouse_source

### ISSUE THREAD

**Issue by kaufmannie** [fivetran/dbt_greenhouse_source#7]
Hello, 
We are looking to exclude the job_office model from our greenhouse integration.  We have no job_office table landed in our data warehouse from our fivetran greenhouse sync. Submitting this issue per the note on the dbt hub (below). Can we get set up to disable the job_office model?

Thanks, 
Annie

https://hub.getdbt.com/fivetran/greenhouse_source/latest/

```
vars:
    greenhouse_using_prospects: false # Disable if you do not use prospects and/or do not have the PROPECT_POOL and PROSPECT_STAGE tables synced
    greenhouse_using_eeoc: false # Disable if you do not have EEOC data synced and/or do not want to integrate it into the package models
    greenhouse_using_app_history: false # Disable if you do not have APPLICATION_HISTORY synced and/or do not want to run the application_history transform model
```
^
Note: This package only integrates the above variables. If you'd like to disable other models, please create an issue specifying which ones.

**Comment by fivetran-jamie** [comment]
Hey thanks for reaching out! Working on adding this right now

**Comment by fivetran-jamie** [comment]
This will be included in v0.2.0 which is getting released today! 

See the source package [PR](https://github.com/fivetran/dbt_greenhouse_source/pull/8) and the transform package [PR](https://github.com/fivetran/dbt_greenhouse/pull/6)
